tests/tracking/test_tracking.py

```diff
@@ -867,56 +867,58 @@ def test_search_runs():
         mlflow.log_param("p2", "aa")
         mlflow.set_tag("t2", "second-tag-val")
 
-    def verify_runs(runs, expected_set):
-        assert set([r.info.run_id for r in runs]) == set([logged_runs[r] for r in expected_set])
-
     experiment_id = MlflowClient().get_experiment_by_name("exp-for-search").experiment_id
 
     # 2 runs in this experiment
     assert len(MlflowClient().list_run_infos(experiment_id, ViewType.ACTIVE_ONLY)) == 2
 
+    def run_and_verify(experiment_ids, query, check, **kwargs):
+        runs = MlflowClient().search_runs(experiment_ids, query, **kwargs)
+
+        assert set([r.info.run_id for r in runs]) == set([logged_runs[r] for r in check])
+
+        # verify output_fromat="list" returns the same as mlflow_client.search_runs
+        fluent_search_runs_run_ids = [
+            run.info.run_id
+            for run in mlflow.search_runs(experiment_ids, query, output_format="list", **kwargs)
+        ]
+        assert fluent_search_runs_run_ids == [run.info.run_id for run in runs]
```

Comment:
I think we can remove this logic now that we're testing `search_runs` behavior in `tests/tracking/fluent/test_fluent.py`. While `search_runs(output_format='list')` produces the same output format as `MlflowClient.search_runs()`, I don't think we need this extra validation logic since we're not explicitly documenting / guaranteeing that these APIs share the same output format.